### DISCUSSION TITLE
failban в docker блокирует веб интерфейс

### DISCUSSION THREAD

**Discussion by bigshoma** [mikopbx/Core#328] [General ( Обо всём )]
debian 11
Установили в docker mikopbx

docker run --cap-add=NET_ADMIN --net=host --name mitats \
           -v /var/spool/mikopbx/cf:/cf \
           -v /var/spool/mikopbx/storage:/storage \
           -v /home/shoma/docker/logrotate.conf:/etc/logrotate.d/docker-contain>
           -e WEB_PORT=81 \
           -e SSH_PORT=23 \
           -e WEB_HTTPS_PORT=82 \
           -e ID_WWW_USER="$(id -u www-data)" \
           -e ID_WWW_GROUP="$(id -g www-data)" \
           -it -d --restart always ghcr.io/mikopbx/mikopbx-x86-64

Когда включаем в веб интерфейсе сетевой экран, то блокирует веб трафик через обратный прокси.
При обращении по IP адресу работает

**Comment by boffart** [comment]
Мб просто доступ к web запрещен в разделе сетевой экран? 
АТС расположена за NAT и доступ разрешили только из локальной сети? 

Наверное следует снять дамп сетевого трафика и смотреть с какого адреса поступают http запросы и разрешить доступ в сетевом экране. 

Я не вижу в этом какой либо ошибки, если не сможете разобраться самостоятельно, то создайте обсуждение:

- https://github.com/mikopbx/Core/discussions
- https://qa.mikopbx.ru/

**Reply by boffart** [reply]
перенес в обсуждения.
